JustSaying.Messaging/Interrogation/InterrogationResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+
+namespace JustSaying.Messaging.Interrogation
+{
+    public class InterrogationResponse : IInterrogationResponse
+    {
+        public InterrogationResponse()
+        {
+            Subscribers = new List<ISubscriber>();
+            Publishers = new List<IPublisher>();
+        }
+
+        public ICollection<ISubscriber> Subscribers { get; }
```

Comment:
@tharris29 I agree with @petemounce. Can you please update IInterrogationResponse to expose IEnumerable<T> instead of ICollection<T>?